File: trunk/PolarMath/src/org/macondo/polar/evaluation/spectrum/HFPercents.java
package org.macondo.polar.evaluation.spectrum;

import org.macondo.polar.data.Training;
import org.macondo.polar.evaluation.Evaluation;

public class HFPercents implements Evaluation<Double> {
	
	public Double evaluate(Training training) {
		double hf = training.evaluate(new HF());
		double tp = training.evaluate(new TP());
		return (hf / tp) * 100;
	}

}

File: PolarMathCS/PolarMath/Evaluation/Spectrum/HFPercents.cs
using PolarMath.Data;
using System;

namespace PolarMath.Evaluation.Spectrum
{
    public class HFPercents : IEvaluation<double>
    {
        public Double Evaluate(SessionData training)
        {
            var hf = training.Evaluate( new HF() );
            var tp = training.Evaluate( new TP() );
            return (hf / tp) * 100;
        }
    }
}
